openfermioncirq/primitives/__init__.py

```diff
@@ -14,6 +14,9 @@
 
 from openfermioncirq.primitives.bogoliubov_transform import bogoliubov_transform
 
+from openfermioncirq.primitives.optimal_givens_decomposition import (
+    front_and_back_givens_decomposition)
```

Comment:
how about calling the function optimal_givens_decomposition instead of front_and_back_givens_decomposition?

Reply:
👍